src/emulator/config.cpp

```diff
@@ -147,7 +148,7 @@ static bool deserialize(Config &cfg) {
     get_yaml_value(config_node, "background-alpha", &cfg.background_alpha, 0.300f);
     get_yaml_value(config_node, "log-level", &cfg.log_level, static_cast<int>(spdlog::level::trace));
     get_yaml_value(config_node, "pref-path", &cfg.pref_path, std::string{});
-    get_yaml_value(config_node, "cpu-backend", &cfg.cpu_backend, 0);
+    get_yaml_value(config_node, "cpu-backend", &cfg.cpu_backend, static_cast<int>(CPUBackend::Unicorn));
```

Comment:
Let it be 0. I do transformation that 0 is Unicorn and 1 is Dynarmic (the enum value is opposite)

Reply:
ah yes i know, i have just set Unicorn because actually Dynarmic is very early, so i can't set it by default ^^
same before on rpcs3 with ppu llvm, not works on all game etc, so set default ppu interpretter, here i set same :)
and is velocity request i using like that and no with magic number like it said ^^

Reply:
is fixed now :)